neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -49,7 +49,7 @@ public UInt256 Hash
 
         InventoryType IInventory.InventoryType => InventoryType.TX;
 
-        public bool IsLowPriority => Type == TransactionType.ClaimTransaction || NetworkFee < Settings.Default.LowPriorityThreshold;
+        public bool IsLowPriority => NetworkFee < Settings.Default.LowPriorityThreshold;
```

Comment:
Can we change it to this?

```c#
public bool IsLowPriority => Type != TransactionType.ClaimTransaction && NetworkFee < Settings.Default.LowPriorityThreshold;
```

Reply:
The problem that I see of not having it in a configuration, is that if it is spam, it is difficult to change. It would be easy to temporarily go to the configuration and disable the `ClaimTransaction` as a priority.

Reply:
I agree Erik, because setting ClaimTransaction as low priority is unfair since it cannot attach fees.